src/modules/endpoint/sagas/watchContentLanguageSwitch.js

```diff
@@ -47,6 +52,9 @@ export function * switchContentLanguage (dataContainer: DataContainer, action: S
     }
     yield put(insert)
   } catch (e) {
+    if (e.message === 'Network request failed') {
+      Alert.alert(t('languageSwitchFailedTitle'), t('languageSwitchFailedMessage'))
```

Comment:
I think alert is not really a good way to indicate a failure.

Best would be if the components react to this.

Reply:
In this special case I think it's fine, because we don't really have a component which could react to the failure. The ChangeLanguageModel just dispatches the action and gets closed immediately after, and the page displayed before is also not responsible imo.

Reply:
This is basically like a `languageNotAvailable` error. So this should somehow modify the state and replace the content with an error.

This can be done later though.

Reply:
No, it's not. LanguageNotAvailable means the language is not used in the city, e.g. lk augsburg has just `de` and `en`, so visiting it with `fr` produces a `languageNotAvailable` error

Reply:
If this should be done otherwise, it is should be done post mortem.

Reply:
Task created